src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -394,31 +394,17 @@ public Symbol()
             ElementResolver = new ElementResolver();
         }
 
-        // TODO - Dynamo 3.0 - use JSONConstructor on this method
-        // and remove custom logic in nodeReadConverter for symbol nodes.
-
         /// <summary>
         ///     Initializes a new instance of the <see cref="Symbol"/> class.
         /// </summary>
+        [JsonConstructor]
```

Comment:
Changing the constructor that is used for Deserialization to another with more arguments will just work ?

Reply:
In this case yes. It was a hack in the serializer (now removed as part of this change) that made this happen before without signature changes.

Reply:
@aparajit-pratap FYI